.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+  build-chain:
+    runs-on: ubuntu-latest
+    name: Build Pull Request
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Chain
+      id: build-chain
+      uses: Ginxo/github-action-build-chain@master
+      with:
+        parent-dependencies: 'optaweb-employee-rostering'
```

Comment:
Why not `parent-dependencies: 'optaplanner'`? This project doesn't depend on optaweb-employee-rostering.

Reply:
I agree this dependency definition is not correct, this comes only from the [repository-list.txt](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/script/repository-list.txt). The logical dependency is optaplanner as @yurloc suggested

Reply:
this is something pending from my side. It also depends on kie-wb-common, for instance. Just trying to reproduce the same behavior we have in Jenkins and once it is stable I will add the proper parent dependencies. Do you agree?

Reply:
Yes, that makes sense. Let's first reproduce the original process and then optimize it.